pages/_document.tsx

```diff
@@ -63,21 +77,13 @@ export default class MyDocument extends Document {
 
           {(this.props as any).emotionStyleTags}
 
-          {/* Google Analytics */}
-          <Script strategy="afterInteractive" src="https://www.googletagmanager.com/gtag/js?id=GTM-5SPM3GH" />
-          <Script id="google-analytics" strategy="afterInteractive">
-            {`
-          window.dataLayer = window.dataLayer || [];
-          function gtag(){dataLayer.push(arguments);}
-          gtag('js', new Date());
- 
-          gtag('config', 'GTM-5SPM3GH');
-        `}
-          </Script>
-          {/* End Google Analytics */}
-
         </Head>
         <body>
+          <noscript
+            dangerouslySetInnerHTML={{
```

Comment:
For this ESLint error, use the eslint-disabl marking that you can see on line 117 to allow this.

Reply:
Done